Makefile.am

```diff
@@ -96,3 +96,8 @@ include $(top_srcdir)/makefiles/test-cases.mak
 
 include $(top_srcdir)/makefiles/help.mak
 
+if HAVE_RST2MAN
+%.1.in: %.1.in.rst
```

Comment:
Wouldn't it be reasonable to assume that someone checking out from Git either has (or can install) the tool, or if not, that file just won't be generated?

For the proper release source tarball, it would of course include the generated file so nobody would need the tool unless they wanted to edit the RestructuredText file.